test/utils/versions/increaseFromLocalVersion.test.ts

```diff
@@ -20,7 +20,8 @@ describe("increaseFromLocalVersion", function () {
   this.timeout(60 * 1000);
 
   const dnpName = "admin.dnp.dappnode.eth";
-  const manifest = {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const manifest: any = {
```

Comment:
Why type any?

Reply:
omit invalid value to cast the type to a valid `Manifest`